src/device.rs

```diff
@@ -599,7 +602,7 @@ impl<B: UsbBus> UsbDevice<'_, B> {
                             let index = StringIndex::new(index);
                             classes
                                 .iter()
-                                .find_map(|cls| cls.get_string(index, lang_id))
+                                .find_map(|cls| cls.get_string(index, lang_id.ok()))
```

Comment:
I think defaulting to EN_US for the lang ID is probably a good idea. I don't think Lang ID is even defined in the USB spec anymore.